test/e2e/packetcapture_test.go

```diff
@@ -394,23 +472,45 @@ func testPacketCaptureBasic(t *testing.T, data *TestData, sftpServerIP string, p
 					IPFamily: v1.IPv4Protocol,
 				},
 				crdv1alpha1.CaptureDirectionSourceToDestination,
+				crdv1alpha1.CaptureLocationBoth,
 				packetCaptureSourcePod(data.testNamespace, clientPodName),
 				packetCaptureDestinationPod(data.testNamespace, tcpServerPodName),
 			),
 			expectedStatus: crdv1alpha1.PacketCaptureStatus{
 				NumberCaptured: 5,
-				FilePath:       getPcapURL("ipv4-icmp"),
+				FilePath:       getPcapURL("ipv4-icmp", "Destination", antreaAgTCPPod),
+				FilePaths:      []string{getPcapURL("ipv4-icmp", "Destination", antreaAgTCPPod), getPcapURL("ipv4-icmp", "Source", antreaAgClientPod)},
```

Comment:
Can we find a way to not make changes for the existing cases? If we have too, make it small.
create some sane defaults for them. for example, the location default=source.  and use a function to apply the defaults. for the filePath use a one-for-all function do the transition.
seems everytime we need to made some changes for the existing cases